### PR TITLE
Minor fixes in connect unittest

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
- Fix import of wrong header file
- Improve function coverage
- Reduce log output

### CODE REVIEW

**Comment by andreas-bok-sociomantic** [comment]
Rebased on latest v6.x.x

**Comment by andreas-bok-sociomantic** [comment]
ping

File: tests/unit/connect.c [hunk 1]
```diff
@@ -185,6 +182,9 @@ int main(int argc, char *argv[])
     drizzle_close(con);
   }
 
+  int cxt = 1;
+  drizzle_set_log_fn(con, log_fn_callback, (void*)&cxt);
+  drizzle_set_verbose(con, DRIZZLE_VERBOSE_DEBUG);
```
Comment (ben-palmer-sociomantic) [inline tests/unit/connect.c:190]: Did you mean to change this from `DRIZZLE_VERBOSE_INFO` to `DRIZZLE_VERBOSE_DEBUG` as well?

File: tests/unit/connect.c [hunk 2]
```diff
@@ -39,6 +39,7 @@
 
 #include "tests/unit/common.h"
 #include <libdrizzle-redux/libdrizzle.h>
+
```
Comment (ben-palmer-sociomantic) [inline tests/unit/connect.c]: Is this new blank line needed?

**Review by ben-palmer-sociomantic — COMMENTED** [review]
Couple of minor comments.

File: tests/unit/connect.c (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply tests/unit/connect.c:190]: Yes. Know it looks a bit counter intuitive wrt to the ci message. 
The idea is that logging only is enabled for the last call to `drizzle_quit` but with DEBUG level. I will rephrase the ci msg to make this clear

**Comment by andreas-bok-sociomantic** [comment]
Updated ci msg

File: tests/unit/connect.c (hunk 2 above)
Comment (andreas-bok-sociomantic) [reply tests/unit/connect.c]: No

**Comment by andreas-bok-sociomantic** [comment]
Removed obsolete new line

Previous head was 52b9083, new **HEAD** at eba9d3c.

<details>
<summary>Diff between heads are:</summary>

```diff

 tests/unit/connect.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/unit/connect.c b/tests/unit/connect.c
index e70d362..a527b70 100644
--- a/tests/unit/connect.c
+++ b/tests/unit/connect.c
@@ -39,7 +39,6 @@
 
 #include "tests/unit/common.h"
 #include <libdrizzle-redux/libdrizzle.h>
-
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>

```
</details>

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**